public/app/features/explore/ExploreTimeControls.tsx

```diff
@@ -90,7 +92,7 @@ export class ExploreTimeControls extends Component<Props> {
         isOnCanvas
         {...timePickerCommonProps}
         timeSyncButton={timeSyncButton}
-        isSynced={syncedTimes}
+        isSynced={splitted && syncedTimes}
```

Comment:
Why do we need it? Does it mean we can end up with `syncTimes=true` and `splitted=false`? It looks like incorrect state 🤔

Reply:
When `isSynced` is true, there is a highlight on the bottom of these buttons. When the sync defaults to false, it is only lit up when you perform an action, so in the non split state where the action is not possible, it will never be highlighted.

I still think the highlight to show there is something up with the time is a good idea, but having it highlighted in a non split state doesn't make sense because there's nothing to sync. With it defaulted to true, we can end up with that combination. Maybe we should only set that state variable to 'true' when it's split but just saying the sync is true is much more straightforward for setting a default.

Reply:
Okay, changed it so the state initializes to false still, we do not add the split boolean into the logic for syncing, and when doing an initial load or a history change, it will sync the split if all the ranges match.